content/blog/18-19/chat2.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "State of the Chat"
+created_at: 14-10-2018
+description: De langverwachte beslissing
+author: het Zeusbestuur
+---
+
+Laat ons de **tl;dr** eerst geven: vanaf nu is [Mattermost](mattermost.zeus.gent) het officiële chat-kanaal van Zeus WPI. Het Slackteam blijft echter wel bestaan voor zij die willen.
+
+# Waarom veranderen?
+
+In het begin van de vakantie hebben we al een [blogpost](/blog/18-19/chat/) gemaakt waarin we zeggen waarom we willen afstappen van ons huidig chatplatform. De grote tekortkomingen waren, kort samengevat:
+
+- Een gebrek aan backlog: je kunt enkel de laatste 10.000 berichten bekijken
+- Een gebrek aan 'hackbaarheid': Slack is closed-source en niet self-hosted, dus wij hangen voor 100% af van het bedrijf achter Slack. Op de gratis versie is het aantal integraties ook beperkt, waardoor we zelf geen nieuwe coole features konden toevoegen.
+
+# Waarom Mattermost?
+
+
+Er zijn veel alternatieven uitgeprobeerd, maar van alle applicaties was Mattermost die waar we het meest tevreden van waren.
+Hoewel Mattermost zichzelf profileert als een alternatief voor Slack, wordt in het totaalpakket niet altijd hetzelfde niveau van bereikt. 
+Alle belangrijke functionaliteiten zijn aanwezig, maar het is soms nog wat "rough around the edges".
+
+De voordelen van Mattermost wegen wel op tegen de nadelen. Het doorslaggevende argument om over te schakelen was het bestaan van een **backlog**, waarvan het ontbreken in Slack voor een aantal zaken (projectwerking, bestuur) heel ongemakkelijk was.
+In combinatie met de openheid van de software lijkt Mattermost uiteindelijk een betere fit voor Zeus als organisatie.
+
+Uit de rondvraag bleek ook dat uit de "open" applicaties, de meeste leden Mattermost zien als een geschikt alternatief. Van andere organisaties hebben we ook gehoord dat die tevreden zijn van Mattermost.
+
+<%= figure 'https://zeus.ugent.be/zeuswpi/1xQrObwH.png', 'De resultaten van de poll uit onze vorige blogpost, per deelnemer waren meerdere antwoorden mogelijk. Hoewel Slack het meeste stemmen heeft, maakt de vrijheid die we krijgen bij Mattermost de #1 van het bestuur.' %>
+
+# Wat gebeurt er met Slack?
+
+Wij kunnen (en willen) niemand forceren om ons in die keuze te volgen. Het bestuur zal voor alle chatting migreren naar Mattermost, en hopen dat jullie dat samen met ons ook doen. Nieuwe leden zullen we ook naar daar verwijzen.
+
+Als bestuur hebben we geen probleem met het voortbestaan van Slack. Idealiter wordt een nieuwe account aanmaken daar afgesloten, en migreert iedereen van zelf, maar op zich heeft nudded als primary owner de controle over wat er met Slack gebeurt.
+
+# Hoe stap je over?
+
+1. Ga naar de mattermost server <https://mattermost.zeus.gent> of <https://chat.zeus.gent>
```

Comment:
kan je al terug accounts aanmaken op chat.zeus.gent?

Reply:
Op dit moment is chat.zeus.gent een redirect naar mattermost, dus ik denk van wel?